tests/bayesian/test_baysian_cp_full.py

```diff
@@ -14,10 +14,14 @@ def setUp(self):
         L2Overrides = {
             "learningRate": 0.1,
             "noise": 1e-8,
-            "cellCount": 256,  # new: 256 # original: 4096
+            "cellCount": 16, # 256,  # new: 256 # original: 4096
             "inputWidth": 8192,  # new: 8192 # original: 16384 (?)
             "sdrSize": 5,
-            "activationThreshold": 0.01,
+            "activationThreshold": 0.5,
+            "forgetting": 0.01,
+            "useSupport": True,
+            # "useProximalProbabilities": True,
```

Comment:
Is there a reason why this is commented out an not set to a particular value?

Reply:
That's the used test configuration. Tested with both configs, could be set to False alternatively